feat: rewrite doc (#40)
argos-ci/docs: docs/get-started/installation.mdx

---

import githubRepoAccess from "@site/static/img/github-repo-access.png";

# Getting started with Argos

> Argos takes less than 10 minutes to install. Follow these steps to setup it on your project.

Reviewer: ```suggestion
Argos takes less than 10 minutes to install. Follow these steps to set it up into your project.
```
